packages/desktop-client/src/components/autocomplete/PayeeAutocomplete.js

```diff
@@ -70,37 +69,41 @@ function PayeeList({
     <View>
       <View
         style={[
-          { overflow: 'auto', padding: '5px 0' },
-          !embedded && { maxHeight: 175 },
+          { overflow: 'auto', padding: '5px 0', ...styles.altMenuText },
+          !embedded && { maxHeight: styles.altMenuMaxHeight },
         ]}
       >
         {createNew && (
+          // "Create payee" menu item
           <View
             {...(getItemProps ? getItemProps({ item: createNew }) : null)}
             style={{
               flexShrink: 0,
               padding: '6px 9px',
               backgroundColor:
-                highlightedIndex === 0 ? colors.n4 : 'transparent',
+                highlightedIndex === 0
+                  ? colors.altMenuItemBackgroundHover
+                  : colors.altMenuBackground,
+              color:
+                highlightedIndex === 0
+                  ? colors.altMenuItemTextHover
+                  : colors.altMenuItemText,
               borderRadius: embedded ? 4 : 0,
             }}
           >
             <View
               style={{
                 display: 'block',
-                color: colors.g8,
```

Comment:
I'd prefer not but could be convinced. This and the split transaction field had "special" colors, inconsistent with the rest of the app. We'd have to make another text color or reuse the colors.noticeText but there is no guarantee it will work across the multiple backgrounds we now have, since I re-introduced the alternate menu background color for all the autocomplete's

Reply:
Ok, that makes sense! If you do end up switching back to the old styling for the “Split transaction” button I think it would make sense to have a new color dedicated to the two elements but it does feel like a bit much to have a special color for just here, I agree.